### ISSUE TITLE
Icons remain in drag mode after letting go

### ISSUE THREAD

**Issue by marcogiannini** [amirzaidi/Launcher3#411]
When I hold down an icon in a folder, the selected application icon remains locked on the screen. Here is a video that explains the problem:
 
https://youtu.be/pyh2JCHHpRA

- Device: Lenovo K6 (aka Lenovo K33a48)
- Android version: 7.0
- Launcher version: Rootless Pixel Launcher 3.3
- Rom: Stock Rom

**Comment by amirzaidi** [comment]
Does this always happen?

**Comment by marcogiannini** [comment]
yes

**Comment by amirzaidi** [comment]
Does this also happen in older versions of this launcher? What about the real Pixel Launcher (older versions)?

**Comment by Znelle** [comment]
I also have this exact same issue as described. It has been like this since I have installed the 2.0 version. It could also be from an earlier version. 
I use an Lenovo P2a42 running Android 7.0.
I also tried to replicate the bug in another device I have, the Samsung A5 2017, but couldn't do it.
For me, the bug is only present when you try to drag an icon from a folder, it stays there the whole time. The only way I could get it to go away, was to force close the app.

**Comment by marcogiannini** [comment]
It also happened with the old releases. The bug also appears on other launchers based on Rootless Pixel Launcher

**Comment by amirzaidi** [comment]
Can you guys try doing this on the latest alpha: https://t.me/appforks

**Comment by marcogiannini** [comment]
Same problem https://youtu.be/8m0B1eP3uKY

**Comment by Znelle** [comment]
Yes, with that patch, it doesn't happen anymore. Everything still works and has full functionality but the bug is gone. Icons don't get stuck anymore. I tried doing all sorts of things but the problem seems to be resolved.
Thank you so much. This was the only real bug I discovered and now it is gone. This is fantastic! 
Love your development work and the launcher.

**Comment by Znelle** [comment]
Edit: spoke too soon. It still gets stuck, but it only happens the second time when you use the launcher. The first time everything went well.

**Comment by amirzaidi** [comment]
I can't reproduce this anymore, is there any secret to this bug that I am unaware of?

**Comment by Znelle** [comment]
I have another video that you might help with finding the issue: https://youtu.be/BASsvxLM4ic
Maybe it has something to do with the brand, being an Lenovo phone. That's the only similarly I can see between our two phones. Maybe it has something to do with longpressing in a folder, because I don't think I had the bug in v1. Also on my Samsung device, this problem isn't present.
Both my Lenovo and Samsung device are running android 7.0
Sometimes the icons don't get stuck when you are doing a sideways motion in combination with longpressing, as you can see in the video with F1 2016, but most of the time this issue occurs.

**Comment by Znelle** [comment]
With longpressing in V1 I mean the pop up notification.

**Comment by amirzaidi** [comment]
I'll need others to chime in, can anyone reproduce this on a non-Lenovo phone?

**Comment by rajatkantinandi** [comment]
I am also having the same problem on lenovo k6 note & the patch didn't solve it

**Comment by tigospires** [comment]
It is true! I'm having the same problem everyone

**Comment by Lgmrszd** [comment]
Hello. I have this bug too. It happens all the time but only on specific conditions:
App icon must be inside a folder.
~~App should be open using entry in an icon menu (accessible with a long tap).~~ After restarting the launcher, this would happen with any app icon, you just need to hold it.

Device info:
- Device: Xiaomi Redmi 9T
- Android version: 10
- Launcher version: 3.9.1
- Rom: Stock Rom

**Comment by sonar98** [comment]
I never had this problem with my Xiaomi MI8 but today I reinstalled my phone and now I'm also having this problem.

Changing the shape of the icons in the settings or restarting the launcher removes the icon(s) that will stay visible
